interop/proto/mls_client.proto

```diff
@@ -229,12 +229,21 @@ message GroupContextExtensionsProposalRequest {
   repeated Extension extensions = 2;
 }
 
+message ProposalDescription {
```

Comment:
It would be useful here to add some comments to document:
* Which proposals can be described this way
* Which fields here go with which proposal types